cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,27 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = ["classmethod", "staticmethod", "abstractmethod"]
+
+    def visit_FunctionDef(self, node):  # noqa
+        decorator_candidates = [x for x in node.decorator_list if \
```

Comment:
And just so I'm clear, when I say "line break" I mean the "\" character. We definitely need to split that comprehension over multiple lines, but inside square brackets (and other matched punctuation) Python doesn't require the line continuation character.